client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts

```diff
@@ -202,7 +216,7 @@ export const useInboundShipmentColumns = () => {
         },
       ],
       {
-        label: 'label.cost-per-unit',
+        label: 'label.cost-price',
         key: 'costPricePerUnit',
         align: ColumnAlign.Right,
         width: 120,
```

Comment:
Not your changes but i think should fix, cost price looks right when group by item is on:
![Screenshot 2025-01-07 at 4 48 32 PM](https://github.com/user-attachments/assets/9bf6a3f4-2dd7-4358-9bb3-32410ad1a358)

But wrong when its off!
![Screenshot 2025-01-07 at 4 48 37 PM](https://github.com/user-attachments/assets/4ccaed4d-30b6-4368-a175-e317f0e2cf55)